Modules/IO/ImageBase/include/itkIOTestHelper.h

```diff
@@ -69,13 +73,16 @@ class IOTestHelper
 
   template <typename ImageType, typename ImageIOType>
   static void
-  WriteImage(typename ImageType::Pointer & image, const std::string & filename)
+  WriteImage(typename ImageType::Pointer & image,
+             const std::string &           filename,
+             typename ImageIOType::Pointer imageio = nullptr)
   {
 
     using WriterType = itk::ImageFileWriter<ImageType>;
     typename WriterType::Pointer writer = WriterType::New();
 
-    typename ImageIOType::Pointer imageio(ImageIOType::New());
+    if (imageio.IsNull())
+      imageio = ImageIOType::New();
```

Comment:
IIRC ITK style requires mandatory braces, even for single line `if`s.  There are a few other instances too.